src/components/Navbar.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/prop-types */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Please do not disable linters. They exist to help you adhere to strict guidelines 👍